x/evm/client/cli/query.go

```diff
@@ -335,6 +336,30 @@ func GetCmdCommand(queryRoute string) *cobra.Command {
 	return cmd
 }
 
+// GetCmdBurnerInfo returns the query to get the burner info for the specified address
+func GetCmdBurnerInfo(queryRoute string) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "command [chain] [burner address]",
```

Comment:
```suggestion
		Use:   "deposit-info [chain] [deposit address]",
```
Reused command
Also, I think we need to be consistent about using one of deposit address or burner address. I like the former more.

Reply:
but the thing is, the name of the protobuf is already "burnerInfo". I am keeping the name of the param, but not the command